README.md

```diff
@@ -20,13 +20,16 @@ Now xboxpy should be installed and ready for use!
 
 ### Use
 
-* All stuff is internally imported by the `xboxpy` module. So: `import xboxpy`
-* You can define the interface you want to use using environment variable 'XBOX_IF':
-  * 'XBDM' (default)
-  * 'nxdk-rdt`
-  * 'gdb' (auto-detected if the `gdb` module exists)
-* Some interfaces will also allow you to specify the target Xbox using the 'XBOX' environment variable ('Host:Port')
+All code is internally imported by the `xboxpy` module.
+So all you have to do is: `import xboxpy`.
 
+You can choose the interface you want to use using environment variable 'XBOX_IF':
+
+  * 'XBDM' (default) - part of the official Xbox Development Kit Debug Kernel by Microsoft.
+  * 'nxdk-rdt` - [nxdk-rdt is an open-source Xbox Remote Dev Tool](https://github.com/XboxDev/nxdk-rdt).
```

Comment:
Diffing quotation marks (' and `)

Reply:
Thanks, I want to fix this before merge :)